src/document_handler.rb

```diff
@@ -0,0 +1,32 @@
+require 'telegramAPI'
+require 'open-uri'
+
+class DocumentHandler
+  attr_reader :document, :api, :url
+
+  def initialize(request_handler)
+    @token = ENV['TOKEN']
+    @document = request_handler.message.document
+
+    @api = TelegramAPI.new(@token)
+    @url = "https://api.telegram.org/file/bot#{@token}/#{api.getFile(document.file_id)['file_path']}"
+  end
+
+  def translate_document
+    File.open('translated.txt', 'w') { |file| file.write(translate_content) }
```

Comment:
Okay, let's go next.
You want to send file back?
Use `.sendDocument` method to do that. It's parameters are chat id and path to your `translated.txt` file.
Please update your code correspondingly.